lang/levels/standard/text_fre.po

```diff
@@ -0,0 +1,841 @@
+
+#. The Death Clock (lv450)
+#: guitext:120
+msgctxt "In-game message"
+msgid "I have a bad feeling about this. You know you are supposed to kill the little white men right?"
+msgstr "J'ai un mauvais pressentiment à propos de ceci. Vous savez que vous êtes censé tuer les petits hommes blancs, n'est-ce pas ?"
+
+#. The Death Clock (lv450)
+#: guitext:121
+msgctxt "In-game message"
+msgid "Yes, we have some creatures left! I never doubted you for a second. Congratulations!"
+msgstr "Oui, il nous reste quelques créatures ! Je n'ai jamais douté de vous une seconde. Félicitations!"
+
+#. The Death Clock (lv450)
+#: guitext:122
+msgctxt "In-game message"
+msgid "You lost your only creature. You needed it to get claim the tiles around your heart. Hopeless."
+msgstr "Vous avez perdu votre unique créature. Vous en aviez besoin pour réclamer les tuiles autour de votre coeur. Désespéré."
+
+#: guitext:123 guitext:124 guitext:125 guitext:126 guitext:127 guitext:128
+#: guitext:129 guitext:130 guitext:131 guitext:132 guitext:133 guitext:134
+#: guitext:135 guitext:136 guitext:137 guitext:138 guitext:139 guitext:140
+#: guitext:141 guitext:142 guitext:143 guitext:144 guitext:145 guitext:146
+#: guitext:147 guitext:148 guitext:149 guitext:150 guitext:151 guitext:152
+#: guitext:153 guitext:154 guitext:155 guitext:156 guitext:157 guitext:158
+#: guitext:159 guitext:160 guitext:161 guitext:162 guitext:163 guitext:164
+#: guitext:165 guitext:166 guitext:167 guitext:168 guitext:169 guitext:170
+#: guitext:171 guitext:172 guitext:173 guitext:174 guitext:175 guitext:176
+#: guitext:177 guitext:178 guitext:179 guitext:180 guitext:181 guitext:182
+#: guitext:183 guitext:184 guitext:185 guitext:186 guitext:187 guitext:188
+#: guitext:189 guitext:190 guitext:191 guitext:192 guitext:193 guitext:194
+#: guitext:195 guitext:196 guitext:197 guitext:198 guitext:199 guitext:200
+msgctxt "In-game message"
+msgid "Moo"
+msgstr "Meuuh"
+
+#: guitext:201
+msgctxt "Level name"
+msgid "Tymnath Herez"
+msgstr "Tymnath Herez"
+
+#: guitext:202
+msgctxt "Level name"
+msgid "Yami"
+msgstr "Yami"
+
+#: guitext:203
+msgctxt "Level name"
+msgid "Temple Deal"
+msgstr "Temple Deal"
+
+#: guitext:204
+msgctxt "Level name"
+msgid "Grail Run"
+msgstr "Grail Run"
+
+#: guitext:205
+msgctxt "Level name"
+msgid "Thug of War"
+msgstr "Thug of War"
+
+#: guitext:206
+msgctxt "Level name"
+msgid "Wetlands"
+msgstr "Wetlands"
+
+#: guitext:207
+msgctxt "Level name"
+msgid "Desperation"
+msgstr "Desperation"
+
+#: guitext:208
+msgctxt "Level name"
+msgid "Dread Mountain"
+msgstr "Dread Mountain"
+
+#: guitext:209
+msgctxt "Level name"
+msgid "Frozen Legion"
+msgstr "Frozen Legion"
+
+#: guitext:210
+msgctxt "Level name"
+msgid "Shores of Battle"
+msgstr "Shores of Battle"
+
+#: guitext:211
+msgctxt "Level name"
+msgid "My Pet Dungeon"
+msgstr "My Pet Dungeon"
+
+#: guitext:212
+msgctxt "Level name"
+msgid "Ghost in the Machine"
+msgstr "Ghost in the Machine"
```

Comment:
Esprit dans la machine?
Compliqué à traduire.

Reply:
I think its a reference to "The Ghost in the Machine" a 1967 book by Arthur Koestler. I used the German book title for my German translation.

Reply:
@CaptainCosmotic I see, thanks, do you happen to know the french name of that book?

Reply:
in french , the name  of the book is "Le Cheval dans la locomotive". But Ghost in the machine is also a album name, a movie name , etc so idk